Apps/Admin/Server/Services/SupportService.cs

```diff
@@ -55,43 +57,52 @@ public class SupportService : ISupportService
         /// <param name="patientRepository">The injected patient repository.</param>
         /// <param name="resourceDelegateDelegate">The resource delegate used to lookup delegates and owners.</param>
         /// <param name="userProfileDelegate">The user profile delegate to interact with the DB.</param>
+        /// <param name="auditRepository">The injected audit repository.</param>
         public SupportService(
             IMapper autoMapper,
             IConfiguration configuration,
             IMessagingVerificationDelegate messagingVerificationDelegate,
             IPatientRepository patientRepository,
             IResourceDelegateDelegate resourceDelegateDelegate,
-            IUserProfileDelegate userProfileDelegate)
+            IUserProfileDelegate userProfileDelegate,
+            IAuditRepository auditRepository)
         {
             this.autoMapper = autoMapper;
             this.configuration = configuration;
             this.messagingVerificationDelegate = messagingVerificationDelegate;
             this.patientRepository = patientRepository;
             this.resourceDelegateDelegate = resourceDelegateDelegate;
             this.userProfileDelegate = userProfileDelegate;
+            this.auditRepository = auditRepository;
         }
 
         /// <inheritdoc/>
-        public async Task<RequestResult<IEnumerable<MessagingVerificationModel>>> GetMessageVerificationsAsync(string hdid, CancellationToken ct = default)
+        public async Task<PatientSupportDetails> GetMessageVerificationsAsync(string hdid, CancellationToken ct = default)
         {
-            IList<MessagingVerification> dbResult = await this.messagingVerificationDelegate.GetUserMessageVerificationsAsync(hdid).ConfigureAwait(true);
+            IList<MessagingVerification> messagingVerifications = await this.messagingVerificationDelegate.GetUserMessageVerificationsAsync(hdid).ConfigureAwait(true);
+            AgentAuditQuery agentAuditQuery = new(hdid, AuditGroup.BlockedAccess);
```

Comment:
I think we should return & display all agent auditing actions in a table on the page, not just changes to data source access.  Can discuss with @ninosamson.